examples/staticmap/cmdline/main.go

```diff
@@ -37,6 +37,7 @@ var (
 	scale     = flag.Int("scale", -1, "Scale affects the number of pixels that are returned.")
 	format    = flag.String("format", "", "Format defines the format of the resulting image.")
 	maptype   = flag.String("maptype", "", "Maptype defines the type of map to construct.")
+	mapid     = flag.String("mapid", "", "MapId defines the mapid to use.")
```

Comment:
Naming convention for the majority of these is to exclude the map prefix even though they are map values. I think it would make sense and read better to use just `id` instead of `mapi`. 

It would also be nice for the description to talk about what `id` does rather than just letting us know it's a 1:1 pass through to the API.  Maybe this line taken from the API docs: associates a map with a particular style or feature

Reply:
I forgot this PR was still open...

I followed the existing code convention. See maptype above. Besides, it's the name of the variable in the request in the static maps API docs. I didn't want to take the liberty of changing variable names given all the others are 1:1 matches.

As for the description, I figured anyone using library would refer to the API docs if someth